--- a/src/CK2_3D.cpp
+++ b/src/CK2_3D.cpp
@@ -164,6 +164,10 @@
         extern void CKBgfxRasterizerGetInfo(CKRasterizerInfo *info);
         RegisterStaticRasterizer(CKBgfxRasterizerGetInfo, "CKBgfxRasterizer");
 #endif
+#if defined(CKRE_STATIC_GX_RASTERIZER)
+        extern void CKGXRasterizerGetInfo(CKRasterizerInfo *info);
+        RegisterStaticRasterizer(CKGXRasterizerGetInfo, "CKGXRasterizer");
+#endif
 
 #if defined(CKRE_DYNAMIC_RENDER_ENGINE)
         XString moduleName = CKRenderEngineModulePath();
